BabyPatterns/DispatchVC.swift

```diff
@@ -208,37 +208,48 @@ extension DispatchVC: WCSessionDelegate {
                  didReceiveMessageData messageData: Data,
                  replyHandler: @escaping (Data) -> Void) {
 
-        // This is just used as a succesfull communication reply
-        defer { replyHandler(Data()) }
+        // TODO: this should probably be a litle smarter now based on the received message
+        defer { replyHandler(Data()) } // This is just used as a succesfull communication reply
 
         let decoder = JSONDecoder()
-        guard let info = try? decoder.decode(WatchCommunication.self, from: messageData) else { return }
-
-        // TODO: look into if we can get rid of the `.none` case
-        guard info.feedingType != .none else { return }
-
-        // TODO: this needs to be a bit better
-        guard let vm = feedingsVM else { return }
-
-        switch info.action {
-        case .start:
-            vm.feedingStarted(type: info.feedingType, side: info.feedingSide)
-        case .stop:
-            guard vm.feedingInProgress(type: info.feedingType) != nil else { return }
-            vm.feedingEnded(type: info.feedingType, side: info.feedingSide)
-            DispatchQueue.main.async {
-                NotificationCenter.default.post(name: K.Notifications.showSavedFyiDialog, object: nil)
+        if let _ = try? decoder.decode(WatchContextCommunication.self, from: messageData),
+            let feedings = feedingsVM?.feedings,
+            let feedingsData = try? JSONEncoder().encode(feedings)
+        {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/cf824e45d0cbfcf6a9e6dbb6efb41319c512eca4/BabyPatterns/DispatchVC.swift#L218"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>DispatchVC.swift:218</code></td>
    </tr>
  </tbody>
</table>